File: pbweberogwcl/src/app/erogazione/commons/dto/tipo-allegato-dto.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
//java:   it.csi.pbandi.pbservizit.pbandisrv.dto.richiestaerogazione.TipoAllegatoDTO
export interface TipoAllegatoDTO {
  descTipoAllegato?: string;
  idTipoAllegato?: number;
}
//java    it.csi.pbandi.pbservizit.pbandisrv.dto.gestionedatigenerali.TipoAllegatoDTO
export class TipoAllegatoRichiestaErogDTO {
  public descTipoAllegato: string;
  public idTipoDocumentoIndex: number;
  public progrBandoLineaIntervento: number;
  public idDichiarazioneSpesa: number;
  public idMicroSezioneModulo: number;
  public flagAllegato: string;
  public numOrdinamentoMicroSezione: number;
  public idProgetto: number;
  public checked: boolean; //solo frontend
}

File: pbservizit/pbservizit-jar/src/main/java/it/csi/pbandi/pbservizit/pbandisrv/dto/richiestaerogazione/TipoAllegatoDTO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbservizit.pbandisrv.dto.richiestaerogazione;

public class TipoAllegatoDTO implements java.io.Serializable {
	static final long serialVersionUID = 1;

	private Double idTipoAllegato;
	private String descTipoAllegato;
	public Double getIdTipoAllegato() {
		return idTipoAllegato;
	}
	public void setIdTipoAllegato(Double idTipoAllegato) {
		this.idTipoAllegato = idTipoAllegato;
	}
	public String getDescTipoAllegato() {
		return descTipoAllegato;
	}
	public void setDescTipoAllegato(String descTipoAllegato) {
		this.descTipoAllegato = descTipoAllegato;
	}
	
	

	
}
